src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -47,20 +46,30 @@ def ingest_tiff_udf(
 
         :param io_uris: Pairs of tiff input - output tdb uris
         :param config: dict configuration to pass on tiledb.VFS
-        :param workers: Number of threads that will spawn for parallelizing ingestion
         """
 
+        from tiledb import filter as compressors
```

Comment:
Is there a reason we call this `compressors` rather than using `filter`? (Maybe to avoid confusing it with some kind of image filter?) Would be good to leave as a comment.

Reply:
I was inclined to use it as compressor for the reason you mentioned. Filters are somehow keyword in image domain and since we use it mainly for compression and iirc we use these terms interchangeably I thought it would make sense. It is not hard requirement though.

Reply:
Let's use the same argument name as we use in `from_bioimg`...